index.js

```diff
@@ -4772,10 +4769,11 @@ export async function createCsharpBom(path, options) {
       if (dlist?.length) {
         pkgList = pkgList.concat(dlist);
       }
-
-      console.log(`pkgList List Size :: ${pkgList.length}`);
+      if(DEBUG_MODE){
```

Comment:
Please run `npm run lint` which will fix the linting issues automatically. Thank you so much for your patience.

Reply:
I ran the lint. For some reason on my machine its producing error for directory name.

Reply:
Could you share the error?

@setchy any ideas?

Reply:
Os error 123, the filename , directory name or voe label syntax is incorrect

Reply:
That's a strange error. Could you contribute a test repo to add to the repotests? I will take care of the linting errors.